src/router/routes/StudentRoutes.js

```diff
@@ -7,13 +7,15 @@ import { studentRoutes } from '~/router/constants/studentRoutes'
 
 const StudentHome = lazy(() => import('~/pages/student-home/StudentHome'))
 const FindTutor = lazy(() => import('~/pages/find-tutor/FindTutor'))
+const StudentProfile = lazy(() => import('~/pages/student-profile/StudentProfile'))
 
 const StudentRoutes = () => {
   return (
     <Suspense fallback={ <Loader size={ 70 } /> }>
       <Routes>
         <Route element={ <StudentHome /> } index />
-        <Route element={ <FindTutor /> } path={ studentRoutes.navBar.findTutor.route } />
+        <Route element={ <FindTutor /> } path={ studentRoutes.navBar.findTutor.nested } />
+        <Route element={ <StudentProfile /> } path={ studentRoutes.accountMenu.myProfile.nested } />
```

Comment:
And don't forget to change it to the way it was)

Reply:
the same here